src/extension/styling-definitions/styling-definition.js

```diff
@@ -0,0 +1,220 @@
+                type: "string",
+                ref: "label.value.colorExpression",
+                expression: "optional",
+                defaultValue: "",
+                show: (data) =>
+                  (propertyResolver.getValue(data, "label.value.colorType") ?? DEFAULTS.FONT_COLOR_TYPE) ===
+                  "byExpression",
+              },
+            },
+          },
+        },
+      },
+      backgroundColorSection: {
+        component: "panel-section",
+        translation: "Object.OrgChart.ColorLabelExpression",
+        items: {
+          backgroundColorItems: {
+            component: "items",
+            ref: "components",
+            key: "backgroundColor",
+            items: {
+              fontColorWrapperItem: {
+                component: "inline-wrapper",
+                items: {
+                  useColorExpression: {
+                    ref: "backgroundColor.colorType",
+                    width: 9,
+                    component: "dropdown",
+                    defaultValue: DEFAULTS.BACKGROUND_COLOR_TYPE,
+                    options: colorOptions,
+                  },
+                  colorPicker: {
+                    component: "color-picker",
+                    type: "object",
+                    ref: "backgroundColor.color",
+                    width: 3,
+                    dualOutput: true,
+                    defaultValue: DEFAULTS.BACKGROUND_COLOR,
+                    show: (data) =>
+                      (propertyResolver.getValue(data, "backgroundColor.colorType") ??
+                        DEFAULTS.BACKGROUND_COLOR_TYPE) === "colorPicker",
+                  },
+                },
+              },
+              colorExpression: {
+                component: "input-field-expression",
+                type: "string",
+                ref: "backgroundColor.colorExpression",
+                expression: "optional",
+                defaultValue: "",
+                show: (data) =>
+                  (propertyResolver.getValue(data, "backgroundColor.colorType") ?? DEFAULTS.BACKGROUND_COLOR_TYPE) ===
+                  "byExpression",
+              },
+            },
+          },
+        },
+      },
+      cardBorderSection: {
+        component: "panel-section",
+        translation: "Object.OrgChart.CardAppearance",
+        items: {
+          cardBorderItems: {
+            component: "items",
+            ref: "components",
+            key: "border",
+            items: {
+              topBar: {
+                component: "checkbox",
+                ref: "border.top",
+                translation: "Object.OrgChart.TopBar",
+                defaultValue: DEFAULTS.BORDER_TOP,
+              },
+              fullBorder: {
+                component: "checkbox",
+                ref: "border.fullBorder",
+                translation: "properties.border",
+                defaultValue: DEFAULTS.BORDER_FULL,
+              },
+              fontColorWrapperItem: {
```

Comment:
Doesn't really matter, but I guess it should be called something with border

Reply:
Yes, I probably forgot to rename after copy paste. I will fix it.